thanos/charts/templates/query.yaml

```diff
@@ -53,9 +53,15 @@ spec:
         {{- range .Values.thanos.query.additionalArgs }}
         - {{ . }}
         {{- end }}
+        {{/* Technically it is possible to add stores and have the default stores all together.*/}}
+        {{/* Enabling standalone would remove the default stores for an empty store list to be maintained entirely manual.*/}}
```

Comment:
As discussed: This message should also be forwarded to the user. Additionally fail or warn if no store is set.